### PR TITLE
Update README.md for new Cloud9 process

### PR DESCRIPTION
**Pull request by jywarren**
Hi, @ananyo2012, @ryzokuken - trying to adjust/improve the README cloud9 instructions here. How does this look?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ Please read and abide by our [Code of Conduct](https://publiclab.org/conduct); o
 
 This is a quick installation for use with the cloud environment https://c9.io - for more standard, full instructions, see below. 
 
-1. If you have a GitHub account, visit https://c9.io and log in with the GitHub button. (Cloud9 now requires an invite unless you enter a credit card. Email plots-dev@googlegroups.com to ask for a free invite!)
-2. Fork this repository to your own GitHub account, creating a `yourname/plots2` project.
-3. Name your project, then enter `yournameplots2` in the "Clone from Git or Mercurial URL" field, and press **Create Workspace** 
+1. Cloud9 now requires an invite unless you enter a credit card, but there's a workaround -- email plots-dev@googlegroups.com to ask for a free invite to get an account! 
+2. On GitHub, fork this repository to your own GitHub account, creating a `yourname/plots2` project.
+3. Name your project, then, on https://c9.io, enter `https://github.com/publiclab/plots2.git` in the "Clone from Git or Mercurial URL" field, and press **Create Workspace**
```
Comment (ananyo2012) [inline README.md]: I think just mentioning `yourname/plots2` in the url works once you have forked it. Clone should be done from the fork not from the `publiclab/plots2` repo

Comment (jywarren) [inline README.md]: I thought someone had trouble today in Gitter, where it requested an SSH key. Perhaps we need to make a video of this whole process or something? But if you're sure, i can change it back.

File: README.md [hunk 2]
```diff
@@ -32,9 +32,9 @@ Please read and abide by our [Code of Conduct](https://publiclab.org/conduct); o
...
 This is a quick installation for use with the cloud environment https://c9.io - for more standard, full instructions, see below. 
 
-1. If you have a GitHub account, visit https://c9.io and log in with the GitHub button. (Cloud9 now requires an invite unless you enter a credit card. Email plots-dev@googlegroups.com to ask for a free invite!)
-2. Fork this repository to your own GitHub account, creating a `yourname/plots2` project.
-3. Name your project, then enter `yournameplots2` in the "Clone from Git or Mercurial URL" field, and press **Create Workspace** 
+1. Cloud9 now requires an invite unless you enter a credit card, but there's a workaround -- email plots-dev@googlegroups.com to ask for a free invite to get an account! 
+2. On GitHub, fork this repository to your own GitHub account, creating a `yourname/plots2` project.
+3. Name your project, then, on https://c9.io, enter `https://github.com/publiclab/plots2.git` in the "Clone from Git or Mercurial URL" field, and press **Create Workspace** 
 4. In the command line prompt at the bottom of the page, type `. ./install_cloudnine.sh` and press enter.
 5. Enter your username when prompted, and let it set things up.
```
Comment (jywarren) [inline README.md]: what about here -- is it your Github username? Or your cloud9 one (I think that's right). It used to be the same when you could create an account via Github. I think that change to non-github linked accounts is causing some slight issues.

File: README.md (hunk 1 above)
Comment (ananyo2012) [inline README.md]: Ok i see but I think its a different issue. Or perhaps you should give the url of the fork i.e. `https://github.com/yourname/plots2.git`

Comment (jywarren) [inline README.md]: ah yes, the url of the fork is a good idea. Changing.

**Comment by jywarren** [comment]
Made changes and merging. But any additional input is welcome on README improvements!

**Pull request merged**